src/libs/Network.js

```diff
@@ -233,7 +233,7 @@ function processNetworkRequestQueue() {
             .catch((error) => {
                 // When the request did not reach its destination add it back the queue to be retried
                 const shouldRetry = lodashGet(queuedRequest, 'data.shouldRetry');
-                if (shouldRetry) {
+                if (shouldRetry && error.name !== 'AbortError') {
```

Comment:
Can you please move this error name to a constant somewhere since it's referenced in multiple places?